CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.12.0.pre (tbd)
+
+Features:
+
+  - speed up `install` and `update` by using the new gem index (@segiddins, @fotanus, @indirect)
```

Comment:
I feel like "compact" means nothing to anyone except the three of us, but we can put it in if you want!